nvflare/app_common/pt/pt_decomposers.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from io import BytesIO
+from typing import Any
+
+import numpy as np
+import torch
+
+from nvflare.fuel.utils import fobs
+
+
+class TensorDecomposer(fobs.Decomposer):
```

Comment:
Let's move this into app_opt/pt

Reply:
Moved https://github.com/NVIDIA/NVFlare/pull/1283/commits/7ba1abe76b8d955fc48fe49984f579a10b4491cb